pkg/controller/servicebindingrequest/olm.go

```diff
@@ -72,15 +72,20 @@ func (o *OLM) ListCSVOwnedCRDs() ([]unstructured.Unstructured, error) {
 		o.logger.Error(err, "on listting CSVs")
 		return nil, err
 	}
-
 	return o.extractOwnedCRDs(csvs)
 }
 
-type eachOwnedCRDFn func(crdDescription *olmv1alpha1.CRDDescription)
+// eachCRDDescriptionFn function to be called against each CRDDescription in a slice.
+type eachCRDDescriptionFn func(crdDescription *olmv1alpha1.CRDDescription)
 
-// loopCRDDescritions takes a function as parameter and excute it against given list of owned CRDs.
-func (o *OLM) loopCRDDescritions(ownedCRDs []unstructured.Unstructured, fn eachOwnedCRDFn) error {
-	for _, u := range ownedCRDs {
+// loopCRDDescritions takes a list of CRDDescriptions (extracted from "owned") and converts to a
+// actual type instance, before calling out for informed function. This method can return error in
+// case of issues to convert unstructured into CRDDescription.
+func (o *OLM) loopCRDDescritions(
```

Comment:
`loopCRDDescriptions` instead of `loopCRDDescritions` (see the missing `p` in there).

Reply:
Thanks! Fixed.